tests/installation/installation_overview.pm

```diff
@@ -64,6 +64,12 @@ sub run() {
             }
         }
     }
+
+    if (check_screen('inst-overview-bootloader-warning')) {
```

Comment:
`check_screen('inst-overview-bootloader-warning', 0)` should suffice so that we don't waste time.